src/main/kotlin/no/nav/familie/ks/sak/barnehagelister/domene/BarnehagelisteMottattRepository.kt

```diff
@@ -0,0 +1,8 @@
+package no.nav.familie.ks.sak.barnehagelister.domene
+
+import org.springframework.data.jpa.repository.JpaRepository
+import java.util.UUID
+
+interface BarnehagelisteMottattRepository : JpaRepository<BarnehagelisteMottatt, UUID> {
+    fun existsBymeldingId(meldingId: String): Boolean
```

Comment:
```suggestion
    fun existsByMeldingId(meldingId: String): Boolean
```
Må sikkert justere navn på test også